README.md

```diff
@@ -37,6 +37,21 @@ With the detected list of emails admin have different options available:
 3. Revoke the authorization for the list of users using any verification method available.
 4. Check the status of the users in order to know if they are verified or registered.
 
+### Metadata mode
+
+This mode provides extra capabilities over the default processing:
+
+* Reads CSV format with header (copy and paste it from your spreadsheet)
+* Stores all columns except the email as authorization metadata
+
+This enables querying the authorization metadata however fits you best.
```

Comment:
It'd be worth documenting how to do that with an example and a link to PostgreSQL's docs

Reply:
A link to PostgreSQL's docs? what do you mean? how to export a csv from a sql table?

Reply:
I thought people may not be familiar with PostgreSQL JSON operators and it's worth pointing at the docs and show how it could be queried like in https://github.com/coopdevs/decidim-coopcat/issues/2#issuecomment-697664892.